tests/Struct/FindologicProductTest.php

```diff
@@ -396,6 +396,38 @@ public function testProductWithCustomFields(): void
         }
     }
 
+    public function testProductWithMultiSelectCustomFields(): void
+    {
+        $data['customFields'] = ['multi' => [
+            'one value',
+            'another value',
+            'even a third one!'
+        ]];
+        $productEntity = $this->createTestProduct($data, true);
+
+        $productFields = $productEntity->getCustomFields();
+        $customerGroupEntities = $this->getContainer()
+            ->get('customer_group.repository')
+            ->search(new Criteria(), $this->salesChannelContext->getContext())
+            ->getElements();
+
+        $findologicProductFactory = new FindologicProductFactory();
+        $findologicProduct = $findologicProductFactory->buildInstance(
+            $productEntity,
+            $this->router,
+            $this->getContainer(),
+            $this->salesChannelContext->getContext(),
+            $this->shopkey,
+            $customerGroupEntities,
+            new XMLItem('123')
+        );
+
+        $attributes = $findologicProduct->getCustomFields();
+        foreach ($attributes as $attribute) {
+            $this->assertEquals($attribute->getValues(), $productFields[$attribute->getKey()]);
```

Comment:
expected and actual should be switched.

Reply:
Oof. Then I might also do that for the other test, from where I copied that ^^.